sparkbox/apprentices-page-next
# Add 15.0 Apprentices

## Description
This PR adds support for the v15.0 apprentices to the page. Additionally, this PR adds the v14.0 (Angus and Alice) apprentices near the call to action.

Additionally, this PR fixes an issue where the final row of previous apprentices would be misaligned when the final row contained either two or three sections.

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. Visit the [test site](https://deploy-preview-39--sb-apprentices.netlify.app/).
- [ ] Confirm the 15.0 apprentices appear in "Current Apprentices" section.
- [ ] Confirm the 14.0 apprentices appear in "Previous Apprentices" section.
- [ ] Confirm the 2.0 and 1.0 apprenticesare vertically aligned with the 6.0 and 5.0 apprentices column, respectively.

## Validation

The following has been completed by the developer:


### Browser Testing

#### Gold Level Browsers

In these browsers, all content is accessible and design matches exactly. In most cases, bugs should be resolved before merge.

**macOS**


**Windows**


**Mobile**


#### Silver Level Browsers

In these browsers, all content is readable, but design may be inaccurate.

**Windows**


**Mobile**


#### Bronze Level Browsers

In these browsers, our only goal is content readability.

**Windows**
